pyproject.toml

```diff
@@ -96,3 +96,8 @@ run.plugins = ["covdefaults"]
 [tool.mypy]
 show_error_codes = true
 strict = true
+
```

Comment:
inline this configuration please,  as shown here https://github.com/tox-dev/tox/blob/main/pyproject.toml#L163